api-tester/src/main/java/com/revenuecat/apitester/kotlin/PurchaseOptionApi.kt

```diff
@@ -13,7 +13,7 @@ private class PurchaseOptionApi {
         val isBasePlan: Boolean = purchaseOption.isBasePlan
     }
 
-    fun checkGooglePurchaseOption(googlePurchaseOption: GooglePurchaseOption) {
+    fun checkGooglePurchaseOption(googlePurchaseOption: com.revenuecat.purchases.google.GooglePurchaseOption) {
```

Comment:
Is this full import required? or just `GooglePurchaseOption`

Reply:
Oooops... I may have clicked a bad autocomplete in Android Studio 🤦‍♂️